thrift/src/main/thrift/shared.thrift

```diff
@@ -186,6 +186,11 @@ struct Image {
   3: required string mediaId
 
   4: optional string source
+
+  5: optional string suppliersReference
```

Comment:
Could we add a comment about what is `suppliersReference`

Reply:
`suppliersReference` is [optional in Grid](https://github.com/guardian/grid/blob/master/common-lib/src/main/scala/com/gu/mediaservice/model/ImageMetadata.scala#L18) and as far as I can tell its not used by any rendering client. We originally added it to this PR because Composer stores it/sends it to CAPI so we wanted to keep parity, but as its not used I'll remove this field.